Assets/scripts/leaderboard.cs

```diff
@@ -0,0 +1,84 @@
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using Shitty.Networking;
+
+public class leaderboard : MonoBehaviour
+{
+
+    public Text leaderboard_text;
+
+    public List<Score> scores = new List<Score>();
+
+
+    public leaderboard()
+    {
+
+    }
+
+    public void Start()
+    {
+        StartCoroutine(getLeaderboard());
+    }
+
+
+    public IEnumerator getLeaderboard()
+    {
+        Http http = new Http();
+        yield return StartCoroutine(http.Get("/scores"));
+        JSONObject scoresJson = new JSONObject(http.data).GetField("scores");
+
+        // reset the leaderboard we have and add the fresh ones
+        scores.Clear();
+        for (int i = 0; i < scoresJson.Count; i++)
+        {
+            string name = scoresJson[i].GetField("name").ToString();
+            name = name.Substring(1, name.Length - 2); // remove quotes ex "gage" => gage
+            int score = int.Parse(scoresJson[i].GetField("score").ToString());
```

Comment:
This is what it takes to convert and use json in c#.. nasty but that is what it takes when changing to a strictly typed language.